central/role/datastore/datastore_impl.go

```diff
@@ -126,11 +126,9 @@ func (ds *dataStoreImpl) RemoveRole(ctx context.Context, name string) error {
 	if err := sac.VerifyAuthzOK(roleSAC.WriteAllowed(ctx)); err != nil {
 		return err
 	}
-	if err := verifyNotDefaultRole(name); err != nil {
-		return err
-	}
-	// Verify storage constraints.
-	if err := ds.verifyRoleNameExists(ctx, name); err != nil {
+
+	// Verify storage constraints
```

Comment:
nit: not sure default role check counts as "storage constraint". Considering you have a more detailed comment next to function implementation, maybe we could delete this comment altogether.